hr_applicant/models/hr_recruitment.py

```diff
@@ -0,0 +1,296 @@
+# See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class Applicant(models.Model):
+    _inherit = "hr.applicant"
+    _description = "Applicant"
+
+    @api.depends("medical_ids")
+    def _compute_no_of_medical(self):
+        for rec in self:
+            rec.no_of_medical = rec.no_of_medical1 = len(rec.medical_ids.ids)
+
+    @api.depends("prev_occu_ids")
+    def _compute_no_of_prev_occu(self):
+        for rec in self:
+            rec.no_of_prev_occu = len(rec.prev_occu_ids.ids)
```

Comment:
```suggestion
            rec.no_of_prev_occu = len(rec.prev_occu_ids)
```
seems no need to define .ids, check if it works correctly without defining .ids